src/bin/stratisd.rs

```diff
@@ -85,20 +124,45 @@ fn run() -> StratisResult<()> {
                          revents: 0,
                          events: libc::POLLIN,
                      });
-            1
+            2
         }
-        None => 0,
+        None => 1,
     };
 
     // Don't timeout if eventable lets us be event-driven
```

Comment:
The comment and the code don't match here. ```engine_fds_end_idx``` is not equal to 0 always.
So this line is equivalent to:

```let poll_timeout = -1;```

Is that what is wanted?

Reply:
You found a bug in my change, this should be:
`let poll_timeout = if engine_fds_end_idx != 1 { -1 } else { 10000 };`